sdk/servicebus/azure_messaging_servicebus/src/client.rs

```diff
@@ -0,0 +1,552 @@
+// Copyright (c) Microsoft Corporation. All Rights reserved
+// Licensed under the MIT license.
+
+use crate::{
+    common::authorizer::Authorizer, ErrorKind, ReceiveMode, Receiver, Result, Sender,
+    ServiceBusError,
+};
+use azure_core::{credentials::TokenCredential, fmt::SafeDebug, http::ClientOptions};
+use azure_core_amqp::{
+    AmqpConnection, AmqpConnectionApis, AmqpConnectionOptions, AmqpOrderedMap, AmqpSymbol,
+    AmqpValue,
+};
+use std::sync::Arc;
+use url::Url;
```

Comment:
Nit: though rust-analyzer probably did this, try to export from `azure_core` only in case we ever redefine these. That said, we'd just go through and fix them anyway but doing so now helps avoid future potential problems e.g., `Url`, `Uuid`, `Method`, et. al.(?).